{{ cookiecutter.package_name }}/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+## Description
+<!-- Provide a general description of what your pull request does.
+Complete the following sentence and add relevant details as you see fit.
+In addition please ensure that the pull request title is descriptive
+and allows maintainers to infer the applicable subpackage(s). -->
+
+This pull request is to address ...
+
+<!-- If the pull request closes any open issues you can add this.
+If you replace <Issue Number> with a number, GitHub will automatically link it.
+If this pull request is unrelated to any issues, please remove
+the following line. -->
+
+Fixes #<Issue Number>
+
+
+## PR Checklist
+
+- [ ] Check out the [contributing guidelines](https://github.com/astropy/astropy/blob/master/CONTRIBUTING.md) and [code of conduct](https://github.com/astropy/astropy/blob/master/CODE_OF_CONDUCT.md)
+- [ ] Check out the [contributing workflow](http://docs.astropy.org/en/latest/development/workflow/development_workflow.html) ( for a practical example [click here](https://docs.astropy.org/en/latest/development/workflow/git_edit_workflow_examples.html#astropy-fix-example) )
+
+- [ ] Give a detailed description of the PR here.
+- [ ] Document changes in the CHANGES.rst file.
+- [ ] Make sure the docs are up to date.
+- [ ] Add yourself to credits.rst file.
```

Comment:
In astropy core, release manager runs a script to populate this. In fact, it is documented in https://docs.astropy.org/en/latest/development/releasing.html#standard-release-procedure . So this is not necessary here.
```suggestion
```